packages/docgen/README.md

```diff
@@ -0,0 +1,246 @@
+# `docgen`
+
+`docgen` helps you to generate the _public API_ of your code. Given an entry point file, it outputs the ES6 export statements and their corresponding JSDoc comments in human-readable format.
+
+Some characteristics:
+
+* If the export statement doesn't contain any JSDoc, it'll look up for JSDoc up to the declaration.
+* It can resolve relative dependencies, either files or directories. For example, `import default from './dependency'` will find `dependency.js` or `dependency/index.js`
+
+## Installation
+
+Install the module
+
+```bash
+npm install @wordpress/docgen --save-dev
+```
+
+## Usage
+
+```bash
```

Comment:
Rebasing from master seems to have deleted your previous comment, @gziolo. For reference 399629ad2ae8b067aa85f7174d77a8184fa4c9f7 added `bash` syntax to this as well.